go/libraries/doltcore/sqle/logictest/DoltRegressionsJenkinsfile

```diff
@@ -0,0 +1,102 @@
+                                    sh "dolt commit -m \"update dolt sql performance results\""
+                                }
+                            }
+                            sh "dolt sql -r csv -q \'select version, test_file, line_num, avg(duration) as mean_duration, result from dolt_history_nightly_dolt_results where version=\"${DOLT_VERSION}\" group by line_num;\' > nightly_mean.csv"
+                            sh "dolt table import -u nightly_dolt_mean_results nightly_mean.csv"
+                            sh "dolt add nightly_dolt_mean_results"
+                            sh "dolt commit -m \"update dolt sql performance mean results\""
+                            sh "dolt push origin nightly"
+                            sh "dolt checkout regressions"
+                            sh "dolt merge nightly"
+                            sh "dolt add ."
+                            sh "dolt commit -m \"merge nightly\""
+                            sh "dolt push origin regressions"
+                            sh "dolt checkout releases"
+                            sh "dolt sql -r csv -q \'select * from releases_dolt_mean_results;\' > releases_mean.csv"
+                            sh "rm -f regressions_db && touch regressions_db"
+                            sh "sqlite3 regressions_db < ../go/libraries/doltcore/sqle/logictest/regressions.sql"
+                            sh "cp ../go/libraries/doltcore/sqle/logictest/import.sql ."
+                            sh "sqlite3 regressions_db < import.sql"
+                            script {
+                                env.DURATION_REGRESSIONS = sh(returnStdout: true, script: "sqlite3 regressions_db \'select * from releases_nightly_duration_change\' | wc -l | tr -d '[:space:]'")
+                                env.RESULT_REGRESSIONS = sh(returnStdout: true, script: "sqlite3 regressions_db \'select * from releases_nightly_result_change\' | wc -l | tr -d '[:space:]'")
+                            }
+                            sh "if [ $DURATION_REGRESSIONS != 0 ]; then exit 1; fi"
+                            sh "if [ $RESULT_REGRESSIONS != 0 ]; then exit 1; fi"
+                        }
+                    }
+                }
+            }
+        }
+    }
+    post {
+        always {
+            node ("liquidata-inc-ld-build") {
+                cleanWs() // cleanup
+            }
+        }
+        failure {
+            emailext body: "${currentBuild.currentResult}: Job ${env.JOB_NAME} build ${env.BUILD_NUMBER}\n More info at: ${env.BUILD_URL}",
+                to: "$SQL_WATCHER_1",
+                subject: "Jenkins Build ${currentBuild.currentResult}: Job ${env.JOB_NAME}"
+            emailext body: "${currentBuild.currentResult}: Job ${env.JOB_NAME} build ${env.BUILD_NUMBER}\n More info at: ${env.BUILD_URL}",
+                to: "$SQL_WATCHER_2",
+                subject: "Jenkins Build ${currentBuild.currentResult}: Job ${env.JOB_NAME}"
```

Comment:
Tried a bunch to send an email to 2 recipients... may go with `to:` and `cc:`, but `to:` I could only get working with a single email address which is annoying.